pkg/processor/native/utils.go

```diff
@@ -14,14 +16,22 @@ func isOpaque(im image.Image) bool {
 	}
 	// No Opaque() method, we need to loop through all pixels and check manually:
 	rect := im.Bounds()
-	for y := rect.Min.Y; y < rect.Max.Y; y++ {
-		for x := rect.Min.X; x < rect.Max.X; x++ {
-			if _, _, _, a := im.At(x, y).RGBA(); a != 0xffff {
-				return false // Found a non-opaque pixel: image is non-opaque
+	isOpaque := true
+	f := func(start, end int) {
+		for y := rect.Min.Y + start; isOpaque && y < rect.Min.Y+end; y++ {
+			for x := rect.Min.X; isOpaque && x < rect.Max.X; x++ {
+				if _, _, _, a := im.At(x, y).RGBA(); a != 0xffff {
+					isOpaque = false // Found a non-opaque pixel: image is non-opaque
+				}
 			}
 		}
 	}
-	return true // All pixels are opaque, so is the image
+	if config.ConcurrentImageProcessingEnabled() {
```

Comment:
We should change the environement variable in the test to test both branches of this conditional.

Reply:
yeah I wanted to do that but I didn't know how to change the config for the test, can you help me on this one?

Reply:
Sure, here you go!
https://github.com/gojek/darkroom/blob/7f6483ab05f6af7f324e83f2936c51f65ebf47b9/pkg/router/routes_test.go#L19-L26